### PR TITLE
Provide quickstart out of the box

### LINKED ISSUE #174: Provide quickstart on system
We want to provide a portable quickstart to deploy a quick and dirty temBoard infrastructure on an existing PostgreSQL installation.

### Target

``` console
$ pip install easy-temboard
$ easy-temboard --ui user@uihost
$ easy-temboard --agent user@host --ui=https://uihost/
```

### Implementation

- Use Ansible as backend. This will ease portability.
- Use versions from PyPI

cc @julmon

### PR DESCRIPTION
**Pull request by bersace**
Closes #174 

### Test instructions

1. `pip install -e .`
1. `docker run --rm -p 5432:5432 postgres`
1. `PGUSER=postgres PGPASSWORD=postgres PGHOST=0.0.0.0 share/create_repository.sh`
1. `PGUSER=temboard PGPASSWORD=temboard PGHOST=0.0.0.0 ./temboard -c share/temboard.conf`
1. Check https://0.0.0.0:8888/. Login, add agent, etc.

### CODE REVIEW

File: temboardui/configuration.py [hunk 1]
```diff
@@ -44,15 +47,41 @@ def __init__(self, configfile, *args, **kwargs):
             'level': 'INFO'
         }
         self.repository = {
-            'host': 'localhost',
-            'user': 'temboard',
-            'port': 5432,
-            'password': 'temboard',
-            'dbname': 'temboard'
+            'host': os.environ.get('PGHOST', '/var/run/postgresql/'),
+            'port': int(os.environ.get('PGORT', '5432')),
```
Comment (julmon) [inline temboardui/configuration.py]: Should be PGPORT there, right ?

File: share/create_repository.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -eu
+
+SQLDIR=$(readlink -m $0/../sql/)
+
+export PGHOST=${PGHOST-/var/run/postgresql}
+export PGDATABASE=${PGDATABASE-temboard}
+
+if ! psql -d postgres -c "SELECT 'SKIP' FROM pg_user WHERE usename = 'temboard'" | grep -q SKIP ; then
+    psql -ad postgres -awc "CREATE ROLE temboard NOSUPERUSER NOCREATEDB NOCREATEROLE INHERIT LOGIN PASSWORD 'temboard';"
```
Comment (julmon) [inline share/create_repository.sh]: Shall we use an env. variable to let users set a better password ?

**Comment by bersace** [comment]
Updated testing instructions.

**Comment by pgiraud** [comment]
I had to set the `DEV` env. variable to 1 when calling `create_repository.sh` so that I have instance groups created before the agent tries to auto register.
Other than that, it looks good to me.

**Pull request merged**